doc/fulcrum-example-config.conf

```diff
@@ -1013,3 +1013,76 @@ rpcpassword = hunter1
 # useful for admins wishing to integrate Fulcrum with monitoring software.
 #
 #pidfile = /path/to/fulcrum.pid
+
+
+
+#-------------------------------------------------------------------------------
+# Reusable Payment Address (RPA) Options
+#-------------------------------------------------------------------------------
+
+# Enable RPA indexing - `rpa` - DEFAULT: 1 for BCH, 0 for all other coins
+#
+# Whether or not to enable the BCH-specific "RPA" index.
+#
+# See: https://github.com/imaginaryusername/Reusable_specs/blob/master/reusable_addresses.md
+#
+# This index takes ~42M of space currently on mainnet (but may grow to >3GB as
+# the blockchain advances over time).  If this index is enabled, the
+# `blockchain.rpa.*` and/or the `blockchain.reusable.*` RPC methods will be
+# available to clients, and the `server.features` map will contain an "rpa"
+# key to indicate that the server supports RPA.
+#
+# If unspecified, then the RPA index and associated RPCs will only be enabled
+# for BCH, and will be disabled for all coins.
```

Comment:
for all **other** coins.

Would be clearer.

Reply:
True, I meant to say all other coins. Otherwise it's a contradiction. Nice catch, thanks. Will fix asap.